app/serializers/hike_serializer.rb

```diff
@@ -1,5 +1,6 @@
 class HikeSerializer < ActiveModel::Serializer
   attributes :title, :description, :id, :location, :difficulty, :duration, :length, :photo,
-  :directionURL, :routeURL, :duration_category,
-  :dog_friendly, :region, :coordinates
+             :directionURL, :routeURL, :duration_category,
```

Comment:
In general, I try to avoid mixing `snake_case` and `camelCase` within a project since it makes it harder to predict how you should type an attribute name when you need to use it later. It seems like most of the rest of your model attributes are `snake_case`, so I'd convert these to `direction_url`, `route_url`, etc